src/git.md

```diff
@@ -217,8 +217,19 @@ there are no glaring errors.
 Once you're all done fixing the conflicts, you need to stage the files that had
 conflicts in them via `git add`. Afterwards, run `git rebase --continue` to let
 Git know that you've resolved the conflicts and it should finish the rebase.
-Once the rebase has succeeded, you'll want to update the associated branch on
-your fork with `git push --force-with-lease`.
+
+Once the rebase has succeeded, you may want to review the changes between your
+old branch and your new one. You can do that with the `git range-diff master
+@{u} HEAD` command, which can be very helpful when you had a complicated rebase
+and you want to make sure you changed the right things. Unlike in regular Git
+diffs, you'll see a `-` or `+` next to another `-` or `+` in the range-diff
+output. The marker on the left indicates a change between the old branch and the
+new branch, and the marker on the right indicates a change you've committed. So,
+you can think of a range-diff as a "diff of diffs" since it shows you the
+differences between your old diff and your new diff.
```

Comment:
This is in the tutorial section ("how do I rebase at all"). I would prefer to move it to "advanced rebasing".

Reply:
I guess it is a somewhat advanced feature, though I use it on pretty much every rebase. The problem with putting it in the advanced section is I worry people won't see it.

Even though it can be a bit confusing to use the command at first, I think it makes rebasing much easier.

On the other hand, I don't want to overwhelm people and make them think it's necessary before doing any rebasing.

So, do you still think it should move?

Reply:
Yes, I think so. I think the initial guide should be limited to things people strictly need to rebase. I'm not too worried about people not seeing it - or I guess, I think it's not a big deal if they don't.

Reply:
Maybe we could add a link to the range-diff section from the tutorial section so people can look at it if they want?

Reply:
The section is about 2 paragraphs directly below, it will be the next thing you see if you keep reading. I'm not sure why you think it's hard to find.

Reply:
I think I just feel like range-diff has been extremely helpful and I want to make sure people see it, even if it's not that far down. But I'm okay with skipping a link 👍